test/tasks/maintenance/backfill_api_key_scopes_task_test.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class Maintenance::BackfillApiKeyScopesTaskTest < ActiveSupport::TestCase
+  test "#process performs a task iteration" do
+    element = create(:api_key)
+    element.update_attribute!(:scopes, nil)
+    Maintenance::BackfillApiKeyScopesTask.process(element)
```

Comment:
Is there a way to call the task for given collection instead of calling process manually for given element?

Reply:
this is just the test code for the process method

Reply:
Clear, just trying to find out a simple way to make codecov happy, since collection is not accessed in tests and thus not considered tested.

Reply:
added simple test coverage